### ISSUE TITLE
Also add buttons to page header menu

### ISSUE THREAD

**Issue by th3hamm0r** [wagtail/wagtail-localize#626]
Currently, if you are on the edit screen of a page and you want to create a translation in another language, or you just want to sync the translation, you have to get to the page listing of the parent, find the target page and use the "more buttons". We've received reports where customers search for those actions and couldn't find them.

It would be nice if the buttons registered with the "register_page_listing_more_buttons"-hook are also registered with the "register_page_header_buttons" (available since 2.16, [commit](https://github.com/wagtail/wagtail/commit/a0c72b4b206fd12f3e4e9ad57bd287aa3088ccb1), [docs](https://docs.wagtail.org/en/stable/reference/hooks.html#register-page-header-buttons)).

I could provide a PR for that. The only challenge is that since 3.0 ([commit](https://github.com/wagtail/wagtail/commit/8bbf41627b6c0907384145e6322a20d16b77f62c)) the header menu buttons use icons (`icon_name`), so not providing any looks bad (all other entries already have one), but setting them needs some backwards-compatible code. Otherwise it would fail with wagtail < 3.0, but I think I could handle that.

**Comment by zerolab** [comment]
Why not have do something like

```
If WAGTAIL_VERSION >= (3, 0):
  register with icons
if WAGTAIL_VERSION >= (2, 16):
  register without
```

I will have time to review on Friday

**Comment by zerolab** [comment]
Fixed by #628
